dracut.sh

```diff
@@ -1151,6 +1154,17 @@ trap 'exit 1;' SIGINT
 readonly initdir="${DRACUT_TMPDIR}/initramfs"
 mkdir -p "$initdir"
 
+if [[ $cpio_reflink == "yes" ]]; then
+    dracut_cpio="$dracutbasedir/dracut-cpio"
```

Comment:
Please add checks if kernel version is sufficent and libc does support copy_file_range(2).
Not everybody are using glibc :)

Reply:
The new functionality is disabled by default at build and runtime, so my preference would be to document the requirements and leave version checks, etc. up to the user.